source/_components/mercedesme.markdown

```diff
@@ -21,6 +21,10 @@ This component provides the following platforms:
  - Sensors - such as fuel status, service interval, remaining km...
  - Device tracker - to track location of your car
 
+<p class='note warning'>
+  The component can integrate cars out of the European and Africa markets only.  
```

Comment:
Shouldn't this be *Africa**n*** markets?

Reply:
Thank you for your valid comment. I've created a new PR to provide a fix.